ui/app/hooks/useCancelTransaction.js

```diff
@@ -17,7 +17,7 @@ import { getConversionRate, getSelectedAccount } from '../selectors'
  */
 export function useCancelTransaction (transactionGroup) {
   const { primaryTransaction, initialTransaction } = transactionGroup
-  const gasPrice = primaryTransaction.txParams?.gasPrice
+  const gasPrice = parseInt(primaryTransaction.txParams?.gasPrice, 16) < 0 ? '0x0' : primaryTransaction.txParams?.gasPrice
```

Comment:
Is this necessary for cancellation transactions? These should only be possible by using our UI, and I didn't think we allowed a negative gas price there :thinking:

Reply:
I would have to check that, but I'm pretty sure I added this change to allow users to "repair" their extension after getting error. I had to reinstall MM to make it work again, even with other updates, but with this change, when opening MM gas price of pending transaction was changed to 0 (instead of negative number) and it was fixing the problem. Does it make sense?